### ISSUE TITLE
Rails v5 Support

### ISSUE THREAD

**Issue by metaskills** [customink/activerecord-aurora-serverless-adapter#9]
Leverage the `RAILS_VERSION` environment variable to test other Rails versions.

**Comment by metaskills** [comment]
Just pushed a `5-2-stable` branch (https://github.com/customink/activerecord-aurora-serverless-adapter/tree/5-2-stable) which is now green thanks to this commit (https://github.com/customink/activerecord-aurora-serverless-adapter/commit/0cd3448b6f7efd1fbfb4551cf0c9ff43a26c9bed). I'm going to follow the same semver pattern as I did in the SQL Server adapter. Master will become `v6.0.0` and each tracked `x-x-stable` branch of Rails/ActiveRecord will be tracked here. So with the branch above, I just released `v5.0.0` to rubygems and I'm gonna check 5-1-stable now...
